sdk/keyvault/azkeys/client_test.go

```diff
@@ -504,7 +484,7 @@ func TestUpdateKeyPropertiesImmutable(t *testing.T) {
 					},
 					ReleasePolicy: &ReleasePolicy{
 						Immutable:     to.Ptr(true),
-						EncodedPolicy: marshalledPolicy,
+						EncodedPolicy: getMarshalledReleasePolicy(testAttestationURL),
```

Comment:
When we write a test, how do we know we need to use this?

Reply:
Tribal knowledge and observing the prior art, I suppose. Setting a release policy is only necessary when testing key release, which we already do, so I don't expect this helper will be more commonly used.